_i18n/es/resources/user-guides/easiest_buy.md

```diff
@@ -1,66 +1,65 @@
 {% assign version = '1.1.0' | split: '.' %}
-{% include disclaimer.html translated="false" version=page.version %}
-## How to obtain Monero
+{% include disclaimer.html translated="true" version=page.version %}
+## Cómo adquirir Monero
 
-This is a guide to obtain your own Monero as of 20150919. This is perhaps the easiest way to purchase and hold Monero.
+Esta es una guía para obtener tu propio Monero, a 20150919. Este es quizá el método más fácil de comprar y guardar Monero.
 
-####Step 1: Buy Bitcoin
+####Paso 1: Comprar Bitcoin
 
-There are many ways to buy Bitcoin. Perhaps the easiest way is through circle.com. Once you have purchased some Bitcoin, you are ready to buy some Monero! Buying Bitcoin is straightforward. Please goto circle.com and just follow the instructions there.
+Hay muchas formas de comprar Bitcoin. Quizá la forma más fácil es a través de circle.com. Una vez que hayas comprado algo de Bitcoin, estás listo para comprar Monero! Comprar Bitcoin es sencillo. Ve a circle.com y sigue las instrucciones ahí.
 
-####Step 2: Set up a mymonero.com account
+####Paso 2: Configura una cuenta mymonero.com
 
-MyMonero.com is an online wallet for Monero, maintained by Monero Core Developer Ricardo Spagni (fluffpony). It is the easiest wallet to use. Simply go to MyMonero.com and click on the "Create an Account" button.
+MyMonero.com es un monedero online para Monero, mantenido por el desarrollador central de Monero, Ricardo Spagni (fluffypony). Es el monedero más fácil de usar. Simplemente ve a MyMonero.com y haz clic en el botón Create an Account.
 
 ![image1](https://github.com/luuul/monero-site/blob/master/knowledge-base/user-guides/png/easiest_way/01.png)
 
-After clicking the button, you will see your private key. This key is what gives you access to your funds. Never share this key with anyone!
+Después de hacer clic, verás tu clave privada. Esta llave es lo que te dará acceso a tus fondos. ḂNunca compartas esta clave con nadie!
```

Comment:
Hay una "Ḃ" iniciando la segunda oración pegada a la palabra "Nunca" - "ḂNunca".

Reply:
Desgraciado Notepad de Windows jaja, y ni siquiera lo noté.